### ISSUE TITLE
Atari Paddle

### ISSUE THREAD

**Issue by drrak** [libretro/prosystem-libretro#24]
_Google translator:_
Is it possible to emulate Atari Paddle on my gamepad analog stick?

Thanks

RPi2, LibreELEC 7.0.3, Gamestarter 2.7

**Comment by underball** [comment]
No. There are no Paddle games for the 7800, only the 2600. All 7800 games are joystick or light gun games. (not including homebrew games) This emulator never had paddle controls in it initially. This emulator does not use 2600 games. 

This is not a development forum for adding features to this Retroarch port that didn't exist in the original emulator.
